### DISCUSSION TITLE
I want my old feed back

### DISCUSSION THREAD

**Discussion by meghan06** [community/community#66273] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**please give us the old UI back, no reason to fuck up something that was perfectly fine in the first place. SMH**

# give me my old feed back #

# why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github why github #

**Comment by wangrongding** [comment]
💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed
